src/evaluations/data/evaluation_configs.py

```diff
@@ -86,7 +86,7 @@
 # We use the np.array with dtype so as to make sure that the lengths are all
 # integers.
 ADBF_LENGTH_LIST = np.array([100_000, 250_000], dtype=np.int64)
-EXP_ADBF_DECAY_RATE = 10
+EXP_ADBF_DECAY_RATE_LIST = np.array([2, 10], dtype=np.int64)
```

Comment:
Do we care about decay_rate < 10?

Reply:
Here we consider decay rate equal to 2 or 10.

When decay rate = 2, the exp-ADBF and the geo-ADBF config in this framework would be equivalent.
Decay rate = 10 is proposed to be used in a wide range of scenarios.

If other decay rates are needed, we can add them later.